autorest/codegen/templates/MANIFEST.in.jinja2

```diff
@@ -0,0 +1,10 @@
+include *.md
+include LICENSE
+recursive-include tests *.py
+recursive-include samples *.py *.md
+{%- for init_name in init_names %}
+include {{ init_name }}
+{%- endfor %}
+{%- if package_mode == "mgmtplane" %}
+include _meta.json
```

Comment:
I don't think we want to include `_meta.json` as part of the manifest, because I think that means that end users get this as part of their wheel, @lmazuel for his thoughts

Reply:
MANIFEST is `sdist`, not wheel, so they won't get it in wheel anyway. But like I said earlier, this should even be part of the sdist.
https://github.com/Azure/autorest.python/pull/1154#discussion_r810400790